frontend/src/Litepicker.res

```diff
@@ -47,7 +46,7 @@ let make = (~sx as uSx=[], ~startDate=?, ~endDate=?, ~onSelect=?) => {
     let _self = init(options)
 
     None
-  }, [dates])
```

Comment:
Looks like this `dates` value was not used

Reply:
You're right. I honestly don't remember what I was trying to do here.